sklearn/semi_supervised/_label_propagation.py

```diff
@@ -290,6 +290,7 @@ def fit(self, X, y):
             self.n_iter_ += 1
 
         normalizer = np.sum(self.label_distributions_, axis=1)[:, np.newaxis]
+        normalizer = np.array([[1] if x[0] == 0 else x for x in normalizer])
```

Comment:
I think there are more idiomatic numpy ways to do this. Indeed, using np.clip might be sufficient (the replacement does not need to be 1, can be any non zero number).

But `normaliser[normaliser == 0] = 1]` might be sufficient

Reply:
you're right is was sufficient to use `normalizer[normalizer == 0] = 1`